backend/account_v2/custom_auth_middleware.py

```diff
@@ -38,11 +38,13 @@ def __call__(self, request: HttpRequest) -> HttpResponse:
         is_authenticated = auth_service.is_authenticated(request)
 
         if is_authenticated:
+            organization_id = UserSessionUtils.get_organization_id(request=request)
+            if request.organization_id and not organization_id:
```

Comment:
@muhammad-ali-e is this redundant check needed? We perform it already [here](https://github.com/Zipstack/unstract/blob/c7f0ecf377525fb8228ec28f90a7570926ddf432/backend/utils/user_session.py#L14)
```suggestion
            if not organization_id:
```

Reply:
@chandrasekharan-zipstack it will make issues with some request path that doesn''t have organization ID